__tests__/index.test.ts

```diff
@@ -122,7 +125,13 @@ describe('#url([selected])', () => {
     it('should use defaults', () => {
       expect(
         new Oas({
+          openapi: '3.0.0',
+          info: {
+            title: 'testing',
+            version: '1.0.0',
+          },
           servers: [{ url: 'https://example.com/{path}', variables: { path: { default: 'path' } } }],
+          paths: {},
         }).url()
```

Comment:
Fleshed these test specs out so they're valid and we don't need to force type them as `RMOAS.OASDocument`.